tests/isVisible.spec.ts

```diff
@@ -93,9 +93,11 @@ describe('isVisible', () => {
       `,
       methods: {
         add() {
+          // @ts-expect-error
```

Comment:
nit: this is probably some way to type this if you use DefineComponent

Reply:
you mean, let's rewrite this test component to composition api?